tests/ui/lifetimes/tail-expr-lock-poisoning.rs

```diff
@@ -0,0 +1,29 @@
+//@ revisions: edition2021 edition2024
+//@ ignore-wasm no panic or subprocess support
+//@ [edition2024] compile-flags: -Zunstable-options
+//@ [edition2024] edition: 2024
+//@ run-pass
```

Comment:
CI is failing as this test needs unwinding support and upstream forgot to add a `//@ needs-unwind` directive

Reply:
submitted https://github.com/rust-lang/rust/pull/126740 upstream